proposals/0003-remove-var-parameters-patterns.md

```diff
@@ -1,6 +1,6 @@
 # Removing `var` from Function Parameters and Pattern Matching
 
-* Proposal: [SE-0003](https://github.com/apple/swift-evolution/proposals/0003-remove-var-patterns-and-parameters.md)
+* Proposal: [SE-0003](https://github.com/apple/swift-evolution/blob/master/proposals/0003-remove-var-patterns-and-parameters.md)
```

Comment:
Let's fix it a bit more while we're at it:
https://github.com/apple/swift-evolution/blob/master/proposals/0003-remove-var-parameters-patterns.md